package.json

```diff
@@ -27,7 +27,8 @@
   },
   "scripts": {
     "setup": "sh ./scripts/init.sh",
-    "build": "yarn workspace @fishjam-cloud/react-native-client run tsc",
+    "build": "yarn workspace @fishjam-cloud/react-native-client run build",
+    "prepare": "yarn workspace @fishjam-cloud/react-native-client run prepare",
```

Comment:
what does yarn build and yarn prepare do and when should we run it? it should be described in README

Reply:
It just runs `expo-module build` and `expo-module prepare` in react-native-client. 

What would you like to see in the README? It already contains a setup script that uses `yarn build`.
`yarn prepare` is only relevant when releasing.

Reply:
It would be good to see what `yarn prebuild` does and when it's needed. "is only relevant when releasing" - why do we need to run this when releasing? Do we need to run this when first checking out the repo?